src/components/connect.js

```diff
@@ -47,14 +47,15 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
     }
 
     invariant(isPlainObject(mapping), 'Request for `%s` must be either a string or a plain object. Instead received %s', prop, mapping)
-    invariant(mapping.url, 'Request object for `%s` must have `url` attribute.', prop)
+    invariant(mapping.url || mapping.value, 'Request object for `%s` must have `url` (or `value`) attribute.', prop)
+    invariant(!(mapping.url && mapping.value), 'Request object for `%s` must not have both `url` and `value` attributes.', prop)
 
     mapping.equals = function (that) {
       if (this.comparison !== undefined) {
         return this.comparison === that.comparison
       }
 
-      return [ 'url', 'method', 'headers', 'body' ].every((c) => {
+      return [ 'value', 'url', 'method', 'headers', 'body' ].every((c) => {
```

Comment:
Not 100% familiar with the code...  But if you have `value: doTransformationWork(rawValue)` in your mapping, that work will be done every time to produce the final value, so the equality check here doesn't prevent that work from being done?

That being said, seems like the primary use case for the value prop is in a `then` or `andThen` prop...  if it's being used in a 'top level' mapping, then why wouldn't you instead just apply the transformations to plain props directly?

Is that correct or am I missing something?  Going to be using this in .then so not really a problem.  But I guess I can also use my own memoization on that transformation function if I ever needed to.